src/uu/chown/src/chown.rs

```diff
@@ -246,6 +246,19 @@ fn parse_spec(spec: &str, sep: char) -> UResult<(Option<u32>, Option<u32>)> {
     } else {
         None
     };
+
+    if user.chars().next().map(char::is_numeric).unwrap_or(false)
```

Comment:
Are you sure this only happens with numeric? I get this with GNU:
```bash
❯ chown someone: foo
chown: invalid spec: ‘someone:’
```

This whole function could get a little refactor btw, using `split_once` and without `unwrap_or("")` and just keeping the `Option`s around. But I guess that's for another PR :)

Reply:
@tertsdiepraam 

> $ chown someone: foo
> chown: invalid spec: ‘someone:’

because someone doesn't exist ?
`touch foo && chown $(whoami): foo   `
works for me

Reply:
That makes sense!